modules/install-ibm-cloud-configuring-the-install-config-file.adoc

```diff
@@ -0,0 +1,50 @@
+// This is included in the following assemblies:
+//
+// installing_ibm_cloud/install-ibm-cloud-installing-on-ibm-cloud.adoc
+
+[id="configuring-the-install-config-file_{context}"]
+= Configuring the install-config file
+
+The `install-config.yaml` file requires some additional details. Most of the information is teaching the installer and the resulting cluster enough about the available IBM Cloud&#174; hardware so that it is able to fully manage it.
+
+. Configure `install-config.yaml`. Change the appropriate variables to match the environment, including `pullSecret` and `sshKey`.
++
+[source,yaml]
+----
+  apiVersion: v1
+  baseDomain: <domain>
+  metadata:
+    name: <cluster-name>
+  networking:
+    machineCIDR: <public-cidr>
+    networkType: OVNKubernetes
+  compute:
+  - name: worker
+    replicas: 2
+  controlPlane:
+    name: master
+    replicas: 3
+    platform:
+      baremetal: {}
+  platform:
+    baremetal:
+      apiVIP: <api-ip>
+      ingressVIP: <wildcard-ip>
+      provisioningNetworkInterface: <NIC1>
+      provisioningNetworkCIDR: <CIDR>
+      hosts:
+        - name: openshift-master-0
+           role: master
+           bmc:
+             address: ipmi://10.196.130.145?privilegelevel=OPERATOR <1>
+             username: root
+             password: <password>
```

Comment:
If you want to tell users how to get this password the command is 
$ ibmcloud sl hardware detail <id> --output JSON | jq .remoteManagementAccounts[0].password

Reply:
Done.